doc/install/debian.md

```diff
@@ -18,7 +18,7 @@ Debian 7/Ubuntu 12 users need to manually install a more recent version of autom
 If you would like to build Apache Thrift libraries for other programming languages you may need to install additional packages. The following languages require the specified additional packages:
 
  * Java
-	* packages: gradle 
+	* packages: gradle (version 6.9.2)
```

Comment:
I don't know that the Gradle version is a fixed requirement. Can clarify that this was the version tested:

```suggestion
	* packages: gradle (version 6.9.2 is confirmed to work)
```